kubera/authentication/authentication_test.go

```diff
@@ -75,3 +75,91 @@ func TestFetchToken(t *testing.T) {
 		})
 	}
 }
+
+// TestCreateNewUser is to create a new user
+func TestCreateNewUser(t *testing.T) {
+	connection := conn.Get()
+	var tests = map[string]struct {
+		HostName  string
+		FirstName string
+		LastName  string
+		Email     string
+		Password  string
+		IsError   bool
+	}{
+		"No host": {
+			FirstName: "test",
+			LastName:  "testF",
+			Email:     "test@test.io",
+			Password:  "P@ssw0rd",
+			IsError:   true,
+		},
+		"No firstName": {
+			HostName: connection.HostName,
+			LastName: "testF",
+			Email:    "test@test.io",
+			Password: "P@ssw0rd",
+			IsError:  true,
+		},
+		// Without Lastname also create user
+		// "No lastName": {
+		// 	HostName:  connection.HostName,
+		// 	FirstName: "test",
+		// 	Email:     "test@test.io",
+		// 	Password:  "P@ssw0rd",
+		// 	IsError:   true,
+		// },
```

Comment:
If it is creating new user without `lastName` then you can convert this to positive scenario. @bhaskarhc